Tools/peg_generator/Makefile

```diff
@@ -1,4 +1,4 @@
-PYTHON ?= python3.8
+PYTHON ?= python3.9
```

Comment:
Shouldn't this be `../../python`?

Reply:
That should be `../../python` indeed!

I'm now getting an `ImportError: cannot import name 'parse' from 'peg_parser' (unknown location)` though.

Reply:
Same here, it doesn't look like the ../../ solution works. :-(

Reply:
That is because is finding first the C extrension that we include in `Modules`

Reply:
> We need to rename Tools/peg_generator/peg_parser to something else. 

Done.